manager/main.go

```diff
@@ -153,6 +112,50 @@ func (s *server) trialIteration(conf *pb.StudyConfig, study_id string, sCh study
 				}
 				tm.Reset(1 * time.Second)
 			}
+		case <-strtm.C:
+			ret, err := s.GetStoredModels(context.Background(), &pb.GetStoredModelsRequest{StudyName: conf.Name})
+			if err != nil {
+				log.Printf("GetStoredModels Err %v", err)
+			}
+			ts, err := dbIf.GetTrialList(study_id)
+			if err != nil {
+				log.Printf("GetTrials Err %v", err)
+			}
+			for _, t := range ts {
+				tid := t.TrialId
+				tst, err := dbIf.GetTrialStatus(tid)
+				if err != nil {
+					log.Printf("GetTrialStatus Err %v", err)
+					continue
+				}
+				if tst == pb.TrialState_COMPLETED {
+					isin := false
+					for _, m := range ret.Models {
+						if m.TrialId == tid {
+							isin = true
+						}
+					}
+					met := make([]*pb.Metrics, len(conf.Metrics))
+					for i, mn := range conf.Metrics {
```

Comment:
why not put this into `isin` if block?  if `isin` is false, then operation here is basically no-op.